Bio/SeqUtils/MeltingTemp.py

```diff
@@ -404,12 +410,25 @@ def make_table(oldtable=None, values=None):
 
     """
     if oldtable is None:
-        table = {"init": (0, 0), "init_A/T": (0, 0), "init_G/C": (0, 0),
-                 "init_oneG/C": (0, 0), "init_allA/T": (0, 0),
-                 "init_5T/A": (0, 0), "sym": (0, 0), "AA/TT": (0, 0),
-                 "AT/TA": (0, 0), "TA/AT": (0, 0), "CA/GT": (0, 0),
-                 "GT/CA": (0, 0), "CT/GA": (0, 0), "GA/CT": (0, 0),
-                 "CG/GC": (0, 0), "GC/CG": (0, 0), "GG/CC": (0, 0)}
+        table = {
+            "init": (0, 0),
+            "init_A/T": (0, 0),
+            "init_G/C": (0, 0),
+            "init_oneG/C": (0, 0),
+            "init_allA/T": (0, 0),
+            "init_5T/A": (0, 0),
+            "sym": (0, 0),
+            "AA/TT": (0, 0),
+            "AT/TA": (0, 0),
+            "TA/AT": (0, 0),
+            "CA/GT": (0, 0),
+            "GT/CA": (0, 0),
+            "CT/GA": (0, 0),
+            "GA/CT": (0, 0),
+            "CG/GC": (0, 0),
+            "GC/CG": (0, 0),
+            "GG/CC": (0, 0),
+        }
```

Comment:
Since you turned off black formatting for the tables above, you might think about turning it off here, too. However, it's your decision.